WSSiOS/WSSiOS/Source/Presentation/Base/WSSTabBarController.swift

```diff
@@ -108,7 +146,7 @@ final class WSSTabBarController: UITabBarController, UITabBarControllerDelegate
             return true
         }
         
-        if !isLogined && (selectedIndex == 2 || selectedIndex == 3) {
+        if !isLogined && !(selectedIndex < 2) {
```

Comment:
```suggestion
        if !isLogined && (selectedIndex >= 2) {
```
이거 내가 쓴 코든데 좋네여! 근데 이게 더 가독성 좋아보이는데 어떤가요!!

Reply:
인정합니다